pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -384,6 +387,30 @@ func doCreateBackupSchedules(
 		kmsURIs, nil, resultsCh)
 }
 
+func maybeOverrideIncrementalScheduleDetails(ctx context.Context, details *jobspb.ScheduleDetails) {
```

Comment:
It might be better to error and fail rather than overriding with a warning?

Reply:
I don't think we can do that because that would mean `CREATE SCHEDULE` with an `on_previous_running = start` would fail as a whole. Since we do not have a way to express full backup and inc backup schedules separately we still want `start` to apply to the full schedule but want to override what applies to the incremental schedule. As mentioned in my comment above and in the slack thread I think we'll change this to unconditionally set inc schedule to have the `WAIT` option.

Reply:
Right 🤦 In _that_ case, maybe the warning could include that? `on_previous_running=start only applies to full backups. incremental backups will use on_previous_running=wait` or something similar?

Reply:
After an internal chat we decided that we should simplify the user story by making `on_previous_running` a configurable option for only full backup schedules. Incrementals will always wait for previous running jobs to complete before spawning a new incremental job. I've updated the commit message and the code with this new implementation. I don't think we even need to log a warning anymore.